.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Never include this file in git, it's auto-generated by the vscode rock extension. Add it to the gitignore, and rebase to remove it completely from the code.